### ISSUE TITLE
Github Actions failing on commit and PR

### ISSUE THREAD

**Issue by Brianbrifri** [pulumi/actions#11]
Am trying out the github actions:
• I added the example file at `.github/main.workflow`
• I added my secrets to github’s secrets
• I added those secrets to the secret arrays in `main.workflow`
However it’s error-ing out on commit:
```
### STARTED Pulumi Preview (Merged Stack) 23:26:56Z

Pulling image: gcr.io/github-actions-images/action-runner:latest
latest: Pulling from github-actions-images/action-runner
169185f82c45: Pulling fs layer
0ccde4b6b241: Pulling fs layer
d0372f57daa2: Pulling fs layer
165911d108d6: Pulling fs layer
54996bce1de5: Pulling fs layer
165911d108d6: Waiting
54996bce1de5: Waiting
0ccde4b6b241: Verifying Checksum
0ccde4b6b241: Download complete
d0372f57daa2: Verifying Checksum
d0372f57daa2: Download complete
169185f82c45: Verifying Checksum
169185f82c45: Download complete
165911d108d6: Verifying Checksum
165911d108d6: Download complete
54996bce1de5: Verifying Checksum
54996bce1de5: Download complete
169185f82c45: Pull complete
0ccde4b6b241: Pull complete
d0372f57daa2: Pull complete
165911d108d6: Pull complete
54996bce1de5: Pull complete
Digest: sha256:c9bb432ec5ec08ee08b040a9fccacebbbf8a91444dac4721600cf5dca9dae57e
Status: Downloaded newer image for gcr.io/github-actions-images/action-runner:latest
Unable to find image 'pulumi/actions:latest' locally
latest: Pulling from pulumi/actions
743f2d6c1f65: Already exists
977e13fc7449: Pulling fs layer
de5f9e5af26b: Pulling fs layer
0d27ddbe8383: Pulling fs layer
228d55eb5a23: Pulling fs layer
08a6cee74f85: Pulling fs layer
a4f94c473c3e: Pulling fs layer
4c16c080066b: Pulling fs layer
c8ca64cd5dfa: Pulling fs layer
ac54b69c39ba: Pulling fs layer
228d55eb5a23: Waiting
08a6cee74f85: Waiting
a4f94c473c3e: Waiting
4c16c080066b: Waiting
c8ca64cd5dfa: Waiting
ac54b69c39ba: Waiting
0d27ddbe8383: Verifying Checksum
0d27ddbe8383: Download complete
977e13fc7449: Verifying Checksum
977e13fc7449: Download complete
de5f9e5af26b: Verifying Checksum
de5f9e5af26b: Download complete
977e13fc7449: Pull complete
228d55eb5a23: Verifying Checksum
228d55eb5a23: Download complete
a4f94c473c3e: Verifying Checksum
a4f94c473c3e: Download complete
c8ca64cd5dfa: Verifying Checksum
c8ca64cd5dfa: Download complete
4c16c080066b: Verifying Checksum
4c16c080066b: Download complete
ac54b69c39ba: Verifying Checksum
ac54b69c39ba: Download complete
08a6cee74f85: Verifying Checksum
08a6cee74f85: Download complete
de5f9e5af26b: Pull complete
0d27ddbe8383: Pull complete
228d55eb5a23: Pull complete
08a6cee74f85: Pull complete
a4f94c473c3e: Pull complete
4c16c080066b: Pull complete
c8ca64cd5dfa: Pull complete
ac54b69c39ba: Pull complete
Digest: sha256:e5f90dab5542301630923844a4bb00cd98ea881cdb874a4acb39add22f1944e6
Status: Downloaded newer image for pulumi/actions:latest

> grpc@1.21.1 install /github/workspace/node_modules/grpc
Logging in using access token from PULUMI_ACCESS_TOKEN
> node-pre-gyp install --fallback-to-build --library=static_library

[grpc] Success: "/github/workspace/node_modules/grpc/src/node/extension_binary/node-v67-linux-x64-glibc/grpc_node.node" is installed via remote

> @pulumi/aws@0.18.7 install /github/workspace/node_modules/@pulumi/aws
node-pre-gyp WARN Using needle for node-pre-gyp https download 
> node scripts/install-pulumi-plugin.js resource aws v0.18.7


Downloading plugin:  0 B / 56.82 MiB    0.00%
Downloading plugin:  2.39 MiB / 56.82 MiB    4.21% 5s
Downloading plugin:  5.56 MiB / 56.82 MiB    9.79% 4s
Downloading plugin:  8.89 MiB / 56.82 MiB   15.65% 3s
Downloading plugin:  12.24 MiB / 56.82 MiB   21.54% 3s
Downloading plugin:  14.89 MiB / 56.82 MiB   26.20% 3s
Downloading plugin:  18.17 MiB / 56.82 MiB   31.97% 2s
Downloading plugin:  20.77 MiB / 56.82 MiB   36.56% 2s
Downloading plugin:  24.01 MiB / 56.82 MiB   42.25% 2s
Downloading plugin:  26.82 MiB / 56.82 MiB   47.20% 2s
Downloading plugin:  29.41 MiB / 56.82 MiB   51.76% 2s
Downloading plugin:  31.43 MiB / 56.82 MiB   55.32% 1s
Downloading plugin:  37.86 MiB / 56.82 MiB   66.62% 1s
Downloading plugin:  43.43 MiB / 56.82 MiB   76.42%
Downloading plugin:  46.89 MiB / 56.82 MiB   82.53%
Downloading plugin:  50.33 MiB / 56.82 MiB   88.57%
Downloading plugin:  52.94 MiB / 56.82 MiB   93.17%
Downloading plugin:  53.66 MiB / 56.82 MiB   94.44%
Downloading plugin:  56.82 MiB / 56.82 MiB  100.00% 3s

> @pulumi/docker@0.17.0 install /github/workspace/node_modules/@pulumi/docker
> node scripts/install-pulumi-plugin.js resource docker v0.17.0


Downloading plugin:  0 B / 14.16 MiB    0.00%
Downloading plugin:  2.44 MiB / 14.16 MiB   17.20% 1s
Downloading plugin:  5.43 MiB / 14.16 MiB   38.35%
Downloading plugin:  9.89 MiB / 14.16 MiB   69.79%
Downloading plugin:  14.00 MiB / 14.16 MiB   98.86%
Downloading plugin:  14.16 MiB / 14.16 MiB  100.00% 0s

> @pulumi/kubernetes@0.24.0 install /github/workspace/node_modules/@pulumi/kubernetes
[resource plugin aws-0.18.7] installing
[resource plugin docker-0.17.0] installing
> node scripts/install-pulumi-plugin.js resource kubernetes v0.24.0


Downloading plugin:  0 B / 20.12 MiB    0.00%
Downloading plugin:  2.33 MiB / 20.12 MiB   11.58% 1s
Downloading plugin:  5.45 MiB / 20.12 MiB   27.08% 1s
Downloading plugin:  8.45 MiB / 20.12 MiB   41.97%
Downloading plugin:  10.98 MiB / 20.12 MiB   54.56%
Downloading plugin:  17.39 MiB / 20.12 MiB   86.43%
Downloading plugin:  20.12 MiB / 20.12 MiB  100.00% 1s

> protobufjs@6.8.8 postinstall /github/workspace/node_modules/protobufjs
[resource plugin kubernetes-0.24.0] installing
npm WARN aws-typescript@ No description
npm WARN aws-typescript@ No repository field.
> node scripts/postinstall

added 202 packages from 243 contributors and audited 1115 packages in 21.869s
npm WARN aws-typescript@ No license field.

found 0 vulnerabilities

### FAILED Pulumi Preview (Merged Stack) 23:28:50Z (1m54.161s)
```
Everything seems successful but ends up failing

**Comment by chrsmith** [comment]
@Brianbrifri thanks for filing a bug report. I guess GitHub isn't providing you with any additional output to see what the problem is?

For the stack(s) you are trying to update, do you know if the updates actually started? i.e. you see the preview/update on app.pulumi.com? (i.e. did the preview fail because the Pulumi preview failed?)

Regardless I'll see what we can do about providing some more information in our container's output.

**Comment by Brianbrifri** [comment]
Chris,
Correct, that is all the output I received.
And no, the preview did not show up at app.pulumi.com
Let me know if you need anything else from me.

**Comment by cnunciato** [comment]
@Brianbrifri We believe this issue's been fixed with pulumi/pulumi#3036, and will go out with the next release of Pulumi. (It appears the workflow action was suppressing the output of whatever error was occurring.) Feel free to reopen if you see this happen with pulumi/actions v0.17.29 or above.

**Comment by Brianbrifri** [comment]
Thanks! I'll try it out on the new release and let you know
